gcc/cp/contracts.cc

```diff
@@ -191,7 +190,7 @@ lookup_concrete_semantic (const char *name)
   return CCS_INVALID;
 }
 
-/* Compare role and name up to either the NUL terminator or the first
+/* Compare role and name up to either the NULL terminator or the first
```

Comment:
This isn't a typo. NUL is a character, NULL is a pointer-like thing. It would be clearer to say "null byte terminator". There are a few other occurrences of such NUL in GCC sources, and none of them are typos.

Reply:
fixed